diff --git a/ui/include/classes/core/APP.php b/ui/include/classes/core/APP.php
index 8422e82ae13..472b332fabe 100644
--- a/ui/include/classes/core/APP.php
+++ b/ui/include/classes/core/APP.php
@@ -19,7 +19,7 @@
 **/
 
 
-if (version_compare(PHP_VERSION, '8.0.0', '<')) {
+if (version_compare(PHP_VERSION, '7.0.0', '<')) {
 	echo sprintf('Minimum required PHP version is %1$s.', '8.0.0');
 	exit;
 }
diff --git a/ui/include/classes/setup/CFrontendSetup.php b/ui/include/classes/setup/CFrontendSetup.php
index 4de3ca32887..649cbc0b15d 100644
--- a/ui/include/classes/setup/CFrontendSetup.php
+++ b/ui/include/classes/setup/CFrontendSetup.php
@@ -25,7 +25,7 @@
  */
 class CFrontendSetup {
 
-	const MIN_PHP_VERSION = '8.0.0';
+	const MIN_PHP_VERSION = '7.0.0';
 	const MIN_PHP_MEMORY_LIMIT = '134217728'; // 128 * ZBX_MEBIBYTE;
 	const MIN_PHP_POST_MAX_SIZE = '16777216'; // 16 * ZBX_MEBIBYTE;
 	const MIN_PHP_UPLOAD_MAX_FILESIZE = '2097152'; // 2 * ZBX_MEBIBYTE;
